### ISSUE TITLE
Gutenberg is extremely unreliable with JAWS and Internet Explorer 11

### ISSUE THREAD

**Issue by Sorintata** [WordPress/gutenberg#8429]
I tried to do all the steps from the Gutenberg accessibility testing(https://make.wordpress.org/accessibility/gutenberg-testing/) but I got stuck several times.

To start with, I am using the folowing programs:

Windows 10
JAWS 18
Internet Explorer 11
Wordpress 4.9.8
Gutenberg 3.4.0

Problems:

Overal,the gutenberg experience is clumsy and unreliable with Jaws and IE. To be more precise:

- The focus jumps randomly when using tab and shift tab;
- Using the virtual cursor is almost impossible to add a new block that is not in the favorite(or most used) list;
- Using the virtual cursor, sometimes the edit options to edit the block is not opening. You can't move, change font, change color, heading level etc.
- With the virtual cursor off, it is tedious to add a new block. For instance, you tap on add new block, and you can't type in the name of the block, because it brings you back to editing the previous block. Using the arrow keys have the same behavior, your only chance is to browse the block categories by the tab key;
- I was completely unable to create the table from the accessibility test. I can add the table block, I can add and delete before and after rows and colums, but I have no edit box to populate the table with text

These are my findings from a 2 hours test in which time I was able to do about 8 tasks from the list in the article provided at the beginning of this report.

**Comment by designsimply** [comment]
Thank you for testing! In order to make this issue more actionable, we'll want to go through and check to see if each item you've raised has already been reported and if it has not then decide if a new issue is needed. This is something anyone volunteering can help with and I've added the `Needs Testing` label for that purpose.

**Comment by nerrad** [comment]
This will need tested to see if this is still an issue with the changes since the original post.

**Comment by afercia** [comment]
Discussed during today's accessibility team bug-scrub, agreed this needs to be tested again.

**Comment by getsource** [comment]
We discussed this during today's triage session.

@Sorintata, would you mind testing to see if this is still an issue, since there have been a lot of changes and fixes since this issue was created?

Thanks so much!

**Comment by karmatosed** [comment]
I am for now going to close this as there have been some improvements to this over time. We can always reopen if needed.
